scripts/deploy-docs.sh

```diff
@@ -1,18 +1,16 @@
 #!/usr/bin/env bash
 GH_REPO="@github.com/librenms-docs/librenms-docs.github.io.git"
 FULL_REPO="https://${GH_TOKEN}$GH_REPO"
-THEME_REPO="https://github.com/librenms-docs/theme_v2.git"
 
 if [ "$EXECUTE_BUILD_DOCS" != "true" ]; then
     echo "Doc build skipped"
     exit 0
 fi
 
-pip install --user 'jinja2<2.9' mkdocs pymdown-extensions
+pip install --user mkdocs mkdocs-material pymdown-extensions
```

Comment:
mkdocs-material by default needs mkdocs 1.x, mkdocs that's currently being installed in travis is 0.17.5 so I'm not sure this is going to work